### PR TITLE
chromium@127.0.6533.76: Fix hash

### LINKED ISSUE #171: chromium@127.0.6533.76: hash check failed
chromium: 126.0.6478.127 -> 127.0.6533.76
Updating one outdated app:
Updating 'chromium' (126.0.6478.127 -> 127.0.6533.76)
Downloading new version
ungoogled-chromium-127.0.6533.76-1_Win64.7z (97.8 MB) [=======================================================] 100%
Checking hash of ungoogled-chromium-127.0.6533.76-1_Win64.7z ... ERROR Hash check failed!
App:         business/chromium
URL:         https://github.com/macchrome/winchrome/releases/download/v127.6533.76-M127.0.6533.76-r1313161-Win64/ungoogled-chromium-127.0.6533.76-1_Win64.7z
First bytes: 37 7A BC AF 27 1C 00 04
Expected:    7ec93264b1e1b242dc9af162eb0a6a1e9d675691
Actual:      e449c1e8b46b1b528ff5befdeb1b91a94d2672d2

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #171

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### chromium

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
